src/app/GitUI/CommandsDialogs/SettingsDialog/Pages/GitConfigAdvancedSettingsPage.cs

```diff
@@ -41,7 +41,12 @@ protected override void SettingsToPage()
             Validates.NotNull(CurrentSettings);
             foreach (GitSettingUiMapping gitSetting in _gitSettings)
             {
-                gitSetting.MappedCheckbox.Checked = CurrentSettings.GetValue<bool>(gitSetting.GitSettingKey) is true;
+                gitSetting.MappedCheckbox.CheckState = CurrentSettings.GetValue(gitSetting.GitSettingKey) switch
+                    {
+                        "true" => CheckState.Checked,
```

Comment:
https://git-scm.com/docs/git-config#Documentation/git-config.txt-boolean
```suggestion
                        "true" or "yes" or "on" or "1" => CheckState.Checked,
```